indico_zodbimport/modules/badges_posters.py

```diff
@@ -92,7 +168,40 @@ def run(self):
         self._migrate_data(manager)
 
     def _translate_tpl_item(self, item):
-        return {k: item.get(old_k) for k, old_k in ITEM_KEY_MAPPING.viewitems()}
+        result = {}
+        new_type = None
+
+        if 'key' in item:
+            new_type = ITEM_TYPE_MAPPING.get(item['key'])
+        elif 'name' in item:
+            new_type = ITEM_TYPE_MAPPING.get(item['name'])
+
+        item.pop('key', None)
+        item.pop('name', None)
+
+        if new_type is None:
+            self.importer.print_warning(cformat('%{yellow!}Template attribute type unknown'), event_id=self.event_id)
+            return
+
+        result['type'] = new_type
+
+        for old_k, old_v in item.viewitems():
+            if old_k in {'selected', 'textAlignIndex', 'fontFamilyIndex', 'fontSizeIndex', 'styleIndex', 'colorIndex'}:
+                continue
+            else:
```

Comment:
I would just outdent that part instead of using an `else` block - you already continue right above anyawy